iree-requirements-ci.txt

```diff
@@ -10,5 +10,5 @@
 # Uncomment to skip versions from PyPI (so _only_ nightly versions).
 # --no-index
 
-iree-base-compiler==3.1.0rc20241122
-iree-base-runtime==3.1.0rc20241122
+iree-base-compiler==3.1.0rc20241126
```

Comment:
One of the actions is failing on this
https://github.com/iree-org/iree-turbine/actions/runs/12040949027/job/33571834502?pr=249#step:4:99

Ah, because the macOS release build failed to upload with `failed, reason: write ECONNRESET` 🤦 : https://github.com/iree-org/iree/actions/runs/12024344562/job/33519713662

We should
* Change that workflow to not need all packages to exist
* Fix the release upload (this has happened a few times lately)

Reply:
I'll update it with tomorrow's build before landing. Technically the one from today had a bug with this stuff on Python 3.10.